arch/arm64/arch.c

```diff
@@ -63,6 +63,10 @@ void arch_early_init(void)
 
 void arch_init(void)
 {
+
+    int pmcr_mask = 1;
+
+    __asm__ volatile("msr pmcr_el0, %0":: "r"(pmcr_mask):);
```

Comment:
Looks good, but need to move it into the percpu routines, so it also gets initialized on more than just the first cpu.